install_config/install/stand_alone_registry.adoc

```diff
@@ -125,35 +125,75 @@ ifdef::openshift-enterprise[]
 [[registry-quick-installation]]
 === Quick Installation for Stand-alone OpenShift Container Registry
 
-When using the quick installation method to install stand-alone OCR, start the
+. When using the quick installation method to install stand-alone OCR, start the
 interactive installation by running:
-
++
 ----
 $ atomic-openshift-installer install
 ----
 
-Then follow the on-screen instructions to install a new registry. The installation questions will be largely the same as if you were installing a full {product-title} PaaS, but when you reach the following screen:
-
-====
+. Then follow the on-screen instructions to install a new registry. The installation questions will be largely the same as if you were installing a full {product-title} PaaS. When you reach the following screen, be sure to choose `2` to follow the registry installation path.:
++
 ----
 Which variant would you like to install?
 
 
 (1) OpenShift Container Platform
 (2) Registry
 ----
+
+. Specify the hosts that make up your cluster:
++
+----
+Enter hostname or IP address:
+Will this host be an OpenShift master? [y/N]:
+Will this host be RPM or Container based (rpm/container)? [rpm]:
+----
++
+See the xref:../../install_config/install/rpm_vs_contanierized/adoc#install-config-install-rpm-vs-containerized[Installing on Containerized Hosts] topic for information about RPM vs containerized hosts.
+
+. Change the cluster host name, if desired:
++
+----
+Enter hostname or IP address [None]:
+----
+
+. Choose the host to act as the storage host (the master host by default):
++
+----
+Enter hostname or IP address [master.host.example.com]:
+----
+
+. Change the default subdomain, if desired:
++
+----
+New default subdomain (ENTER for none) []:
+----
++
+[NOTE]
+====
+All certificates and routes are created with this subdomain. Ensure this is set to the correct desired subdomain to avoid having to change the configuration after install.
 ====
 
-Be sure to choose `2` to follow the registry installation path.
+. Specify a HTTP or HTTPS proxy, if needed:
++
+----
+Specify your http proxy ? (ENTER for none) []:
+Specify your https proxy ? (ENTER for none) []:
+----
+
+. The next page will summarize your install and start to gather the host information.
```

Comment:
Should this last step really be its own step since this is not a step taken by the user? Seems like the system is working in the background to gather host information.